DataMigration/src/test/java/org/sagebionetworks/dian/datamigration/HmDataModelTests.java

```diff
@@ -221,7 +221,10 @@ public void test_findDeviceId() {
                                 "d1a5cbaf-288c-48dd-9d4a-98c90213ac01", "1576165222"),
                         new HmDataModel.TableRow.ParticipantDeviceId(
                                 "2", "2",
-                                "abc5cbaf-288c-48dd-9d4a-98c90213ac01", "1576165222")
+                                "abc5cbaf-288c-48dd-9d4a-98c90213ac01", "1576165222"),
+                        new HmDataModel.TableRow.ParticipantDeviceId(
+                                "3", "2",
+                                "def5cbaf-288c-48dd-9d4a-98c90213ac01", "1586165222")
```

Comment:
Shouldn't this new record have a distinct participant id (something other than `def5cbaf-288c-48dd-9d4a-98c90213ac01`) and a different time stamp (something other than `1586165222`)?

Reply:
The bug was specifically with there being multiple table entries with the same participant ID, this is the correct test setup.   Also, the timestamps are different in the two entries, 1576165222 vs 1586165222

Reply:
Sorry, may be a miscommunication.  The "2" is the participant ID. The "3" is the table row ID.  And the "def5cbaf-288c-48dd-9d4a-98c90213ac01" is the Device ID.

Reply:
So the four constructor param's are (1) tableId, (2) participantId (aka table row ID), (3) device id, (4) time stamp.  And the new logic for `findParticipantDeviceId ()` is to find the cases in which the same participant has two device IDs and to return the record with the later time stamp.  OK, I get it. It would be "nice" to use a different device ID for the record with timestamp "1586165222" but it suffices that just the time stamp is different.